modules/rules_cc_hdrs_map/0.0.1/presubmit.yml

```diff
@@ -0,0 +1,13 @@
+matrix:
+  platform:
+  - centos7
+  - debian10
+  - ubuntu2004
+  - macos
+  - windows
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    build_targets:
+    - '@rules_cc_hdrs_map//...'
```

Comment:
you probably don't want `//...` here as that descends into packages that load stardoc, and are therefore not part of the public API

Reply:
Thanks for the tip!